services/web/server/src/simcore_service_webserver/projects/projects_api.py

```diff
@@ -275,11 +284,11 @@ async def remove_project_interactive_services(
                 )
 
             # save the state if the user is not a guest. if we do not know we save in any case.
-            with suppress(director_exceptions.DirectorException):
+            with suppress(_DirectorServiceError):
                 # here director exceptions are suppressed. in case the service is not found to preserve old behavior
-                await director_api.stop_services(
+                await stop_services(
                     app=app,
-                    user_id=user_id,
+                    user_id=str(user_id),
```

Comment:
why? user_id is already a string here.... and actually it should be an int

Reply:
That's why, let me fix it properly